cmd/treedb_text_ingest_qual/main.go

```
@@ -0,0 +1,182 @@
+		if err == nil {
+			return fmt.Errorf("multiple JSON values")
+		}
+		return fmt.Errorf("trailing JSON: %w", err)
+	}
+	return nil
+}
+
+type gitResolver func(args ...string) (string, error)
+
+func resolveLocalGit(args ...string) (string, error) {
+	output, err := exec.Command("git", args...).CombinedOutput()
```

golangci-lint (2.12.2)

[medium] G204: Subprocess launched with variable

(gosec)